code/common/analyticslib.q

```diff
@@ -47,7 +47,7 @@ ffill:{[args]
 
 / - General pivot function
 pivot:{[args]
-	checkargs[args;`table`by`piv`var;(" sS")];
+	checkargs[args;`table`by`piv`var`f`g;(" sS")];
  	/ - if user has not specified f or g set to defaults
  	if[not all `f`g in key args;
```

Comment:
I think a change is also needed here so that if only one of `f` _or_ `g` is passed, it will be used. As it stands, if only one is passed, both will be set to their default values so the one that is passed will be overwritten.